scripts/exportrestapi.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+AWS_PROFILE="${AWS_PROFILE:-atat-sandbox-dev}"
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied."
+    echo "Expected Parameters:"
+    echo "  1. The ID of a AWS::ApiGateway::RestApi, for example 'd82arw4bnk'"
+    echo
+    echo "RestApis visible to AWS profile ${AWS_PROFILE}:"
+    aws apigateway get-rest-apis --profile ${AWS_PROFILE} | jq -c '.items[] | (.createdDate |= todate) | {rest_api_id: .id, stack_name: .tags."aws:cloudformation:stack-name", created_date: .createdDate}'
```

Comment:
You should be able to pass the `jq` stuff to the `--query` flag of the AWS CLI?

Reply:
I'm providing the RestApi id options for selection.  Do you mean that I could feed those options into AWS CLI so they can pick?

Reply:
Oh, you mean do away with the `jq` call altogether.

Reply:
Changed in db668a3aba43aef84786a95cb13d304c6f38a527